src/Shell/WorkerShell.php

```diff
@@ -5,21 +5,25 @@
 use Cake\Core\Configure;
 use CvoTechnologies\Gearman\JobAwareTrait;
 use GearmanJob;
-use GearmanWorker;
 use Psr\Log\LogLevel;
 
 class WorkerShell extends Shell
 {
     use JobAwareTrait;
 
+    /**
+     * Process jobs by calling main() function in Shell Tasks.
+     *
+     * @return mixed Instance of Shell Task
+     */
     public function main()
     {
         $worker = $this->gearmanWorker();
 
-        $jobs = Configure::read('Jobs');
+        $jobs = self::getJobs();
 
         foreach ($jobs as $job => $options) {
-            $worker->addFunction($job, function(GearmanJob $job) use ($options) {
+            $worker->addFunction($job, function (GearmanJob $job) use ($options) {
                 $task = $this->Tasks->load($options['className']);
 
                 $workload = @unserialize($job->workload());
```

Comment:
You will need @codingStandardsIgnoreStart/End tag due to `@` usage, else PHPCS will complain.